packages/amplify_auth_cognito/example/lib/Widgets/SignInWidget.dart

```diff
@@ -35,6 +35,17 @@ class _SignInWidgetState extends State<SignInWidget> {
     }
   }
 
+  void _signInWithWebUI() async {
+    try {
+      SignInResult res = await Amplify.Auth.signInWithWebUI(provider:  AuthProvider.google);
```

Comment:
we should add an example here without the provider as well.

Reply:
Done.